src/StackExchange.Redis/ResultProcessor.cs

```diff
@@ -960,12 +960,12 @@ protected override bool SetResultCore(PhysicalConnection connection, Message mes
                                     if (val.IsEqual(CommonReplies.yes))
                                     {
                                         server.ReplicaReadOnly = true;
-                                        Log?.LogInformation($"{Format.ToString(server)}: Auto-configured (CONFIG) read-only replica: true");
+                                        Log?.LogInformationAutoConfiguredConfigReadOnlyReplicaTrue(new(server));
```

Comment:
Can we merge these two and make the true/false a `bool` parameter? I'm fine with the case changing if we use default logging